adafruit_atecc/adafruit_atecc.py

```diff
@@ -45,8 +45,17 @@
 """
 import time
 from struct import pack
+
+# Since the board may or may not have access to the typing library we need
+# to have this in a try/except to enable type  hinting for the IDEs while
+# not breaking the runtime on the controller.
+try:
+    from typing import Any, Sized
+except ImportError:
+    pass
+
 from micropython import const
-from adafruit_bus_device.i2c_device import I2CDevice
+from adafruit_bus_device.i2c_device import I2CDevice, I2C
```

Comment:
The `I2C` that we want to import and use for typing is `busio.I2C` it can imported like `from busio import I2C`.

That import can go up inside of the try/except block for the typing imports, underneath the existing `typing` import.